### PR TITLE
Single video page

### PR DESCRIPTION
**Pull request by sharathnair9999**
Featuring : 

- Single Video Page with Title, Description - dismissable, Channel name and Icon (Responsive)
- Side video cards on single video page.
- Responsive React Player

### CODE REVIEW

**Review by Richa-rani02 — APPROVED** [review]
code looks clean and readable

File: public/index.html [hunk 1]
```diff
@@ -3,9 +3,13 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Akshar:wght@500&family=Fira+Sans&family=Josefin+Sans:wght@300;500;600;700&family=Rubik&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (devananth) [inline public/index.html]: Why *imported* same links **twice** ?

**Review by devananth — CHANGES_REQUESTED** [review]
- Everything seems *good*
- Liked the way you have used *custom hooks* for **document.title**

File: public/index.html (hunk 1 above)
Comment (sharathnair9999) [reply public/index.html]: oops didnt see that. will remove

**Pull request merged**